### PR TITLE
Accordion animation removal

### PR DESCRIPTION
**Pull request by aknobloch3**
- Add accordion--footer to example page
- Remove jquery slideToggle (change to display: block/none)

screen shot 2019-02-07 at 2 30 21 pm

### CODE REVIEW

File: src/sass/modules/accordion/_accordion.scss [hunk 1]
```diff
@@ -40,6 +40,12 @@
     border-top: 2px solid $gray-100;
   }
 
+  .accordion--footer {
```
[the remark this answers is not in the document]
Comment (aknobloch3) [reply src/sass/modules/accordion/_accordion.scss]: Went ahead and extended .card styles into the accordion

**Comment by aknobloch3** [comment]
Here is the focus state of the accordion.
screen shot 2019-02-08 at 8 09 50 am

**Comment by jrauh** [comment]
@aknobloch3 Has the focus been device tested yet?

**Comment by aknobloch3** [comment]
> @aknobloch3 Has the focus been device tested yet?

Nope. But it is just the standard browser focus, not custom.

**Pull request merged**